ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2754,6 +2754,15 @@ router_bgp:
   bgp:
     bestpath:
       d_path: < true | false >
+  # extension of the old dm to allow for multiple listen ranges and addtional filter capabilities
```

Comment:
```suggestion
  # New improved "listen_ranges" data model to support multiple listen ranges and additional filter capabilities
```

Please add comment above the old keys.
```
  # "bgp_listen_range_prefix" and "peer_filter" will be deprecated in AVD v4.0
  # These should not be mixed with the new `listen_ranges` key above to avoid conflicts.